### ISSUE TITLE
X-WAV podcast enclosures don't play.

### ISSUE THREAD

**Issue by pperrin** [RomanSixty/Feed-on-Feeds#28]
(This didn't work in the original either - so its probaby never worked)

It shows a 'click for podcast' image and says (audio/x-wav) but clicking stuff does nothing...

**Comment by plaidfluff** [comment]
Yeah the media/enclosure handling stuff needs to be totally revamped. It should be pretty easy to just replace it with an <audio> tag or whatever.
